wiseconnect/components/common/inc/sl_constants.h

```diff
@@ -0,0 +1,184 @@
+/*
+ * EVALUATION AND USE OF THIS SOFTWARE IS SUBJECT TO THE TERMS AND
+ * CONDITIONS OF THE CONTROLLING LICENSE AGREEMENT FOUND AT LICENSE.md
+ * IN THIS SDK. IF YOU DO NOT AGREE TO THE LICENSE TERMS AND CONDITIONS,
+ * PLEASE RETURN ALL SOURCE FILES TO SILICON LABORATORIES.
+ * (c) Copyright 2018, Silicon Laboratories Inc.  All rights reserved.
+ */

```

Comment:
This looks a different kind of license header than the Zlib and MSLA ones. We also don't have any `LICSENSE.md` file in the tree (or this PR).

Reply:
I have added `license.md`.